doc/JSON_INFO.md

```diff
@@ -4011,6 +4011,7 @@ The contents of use_action fields can either be a string indicating a built-in f
 "use_action": {
     "type": "transform",  // The type of method, in this case one that transforms the item.
     "target": "gasoline_lantern_on", // The item to transform to.
+    "variant_type": "condom_plain", // (Optional) Variant type to set for the transformed item. Special string `<any>` will pick a random variant from all available variants.
```

Comment:
```suggestion
    "variant_type": "condom_plain", // Variant type to set for the transformed item. Defaults to the special string `<any>` that will pick a variant according to the weighted list of all variants. This does nothing for targets without variants.
```

Reply:
I disagree with the wording you've given but I pushed a commit for this. It's already a rather verbose explanation, would like to try to keep it reasonable.